packages/edit-navigation/src/components/layout/index.js

```diff
@@ -171,6 +184,7 @@ export default function Layout( { blockEditorSettings } ) {
 									<ComplementaryArea.Slot scope="core/edit-navigation" />
 								)
 							}
+							secondarySidebar={ renderSecondarySidebar() }
```

Comment:
Could this be simplified?
```suggestion
							secondarySidebar={ isInserterOpened && ( <InserterSidebar /> ) }
```

Reply:
For now...yes. In the future we'll probably want to introduce List View so I'm following the pattern already established in other editors which is to have a dedicate function to determine the secondary sidebar.

I'll simplify for now.

Reply:
✅